kotlinpoet/src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -359,6 +368,23 @@ public class FunSpec private constructor(
       typeVariables += typeVariable
     }
 
+    @DelicateKotlinPoetApi("Context receivers are currently a preview feature")
+    public fun contextReceiver(receiverTypes: Collection<TypeName>): Builder = apply {
+      check(!name.isConstructor) { "$name cannot have receiver type" }
```

Comment:
```suggestion
      check(!name.isConstructor) { "$name: constructors cannot have context receivers" }
```

Reply:
This was copied from the receiver function, so then we would have two error messages for the same thing?

Reply:
Ok fine then I'll change it